sw/snRuntime/src/start_occamy.c

```diff
@@ -0,0 +1,115 @@
+// Copyright 2020 ETH Zurich and University of Bologna.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+#include "snrt.h"
+#include "team.h"
+
```

Comment:
I was wondering whether we maybe can make `start_occamy.[S|c]` the default that also works on the specialized case of the single cluster version, just to avoid bifurcation? Do you see troubles in going there?

I think the only thing that really needs to be adjusted is the `bootdata` struct that is generated from a script for the single-cluster version to indicate that there is just a single cluster, with a single S1 quadrant :).

Reply:
No I think that would make sense to align the two runtimes. Apart from the different `bootdata` struct they are pretty much the same.

How or where exactly is the `bootdata` generated for the `snitch_cluster`?

Reply:
For the cluster it is generated here: https://github.com/pulp-platform/snitch/blob/078a14ab715a759a6146609cea3fba53a3dcf4c2/util/clustergen/cluster.py#L357-L360

For `occamy` we probably want to do something very similar (same template) but [here](https://github.com/pulp-platform/snitch/blob/078a14ab715a759a6146609cea3fba53a3dcf4c2/util/occamygen.py)

Reply:
I have now aligned the two runtimes and the `bootdata` is generated automatically based on the configuration file also for occamy.